sycl/include/sycl/ext/intel/experimental/esimd/detail/math_intrin.hpp

```diff
@@ -574,7 +575,7 @@ __esimd_dpas_inner(const __ESIMD_DNS::vector_type_t<T0, SZ> *src0,
     for (unsigned s = 0; s < systolic_depth; s++) {
       src1_ops_per_dword = 32 / (ops_per_chan * src1_el_bits);
       // U = s / src1_ops_per_dword;
-      U = s >> unsigned(log2(src1_ops_per_dword));
+      U = s >> unsigned(std::log2(src1_ops_per_dword));
```

Comment:
This fix is Ok too, but IMO, it is better to just remove this code/function because ESIMD emulator is deleted.
In this we don't need adding 'include <cmath>' as well.

Reply:
@turinevgeny - what is your opinion in this case?

Reply:
The customer wants the fix ASAP and that might be a little more complicated so I decided to do the trivial fix now and we can do the host code removal whenever.

Reply:
I would agree with Nick.